src/rust/engine/src/context.rs

```diff
@@ -140,6 +142,7 @@ impl Core {
       rule_graph: rule_graph,
       types: types,
       fs_pool: fs_pool.clone(),
+      http_client: reqwest::async::Client::new(),
```

Comment:
Assuming that this doesn't have any threads of its own, it should be fork safe. But would be good to confirm before landing.

Reply:
I checked! It contains an internal threadpool for DNS resolution! Sneaky! Good catch :)